components/dashboard/src/start/StartPage.tsx

```diff
@@ -139,3 +142,16 @@ function StartError(props: { error: StartWorkspaceError }) {
     }
     return <p className="text-base text-gitpod-red w-96">{error.message}</p>;
 }
+
+function OrgDefaultWorkspaceWarningView(props: { workspaceId: string }) {
+    const { data: imageInfo } = useDefaultWorkspaceImageQuery(props.workspaceId);
+    if (imageInfo?.source !== "organization") {
+        return null;
+    }
+    return (
+        <Alert className="w-96 mt-4" type="warning">
+            A custom <span className="font-medium">default workspace image</span> is set for this organization. Contact
+            an organization owner or specify a different image in <code>.gitpod.yml</code>.
+        </Alert>
```

Comment:
**question**: Would you say it makes sense to try to hide the latest editor warning when this warning is shown? Maybe it's too much. Thoughts?

<img width="542" alt="Screenshot 2023-09-21 at 18 34 24" src="https://github.com/gitpod-io/gitpod/assets/120486/bcf0216c-61e1-4322-97c2-11d2c0ae36e8">

Reply:
Addressed https://github.com/gitpod-io/gitpod/pull/18767/commits/88f00b543ed180847cd1f0af3d0c6761f7586ef3